docs/index.md

```diff
@@ -15,46 +15,48 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
-| Lab Array      |          |                                                            |                                                                                                                                                                                                                             |
-| lab.lab_id     | Numeric  | 16 Digit GE ID                                             | Unique GE ID for specific lab result (ObsID)                                                                                                                                                                                |
-| lab.type       | Text     | BLL                                                        | Static BLL unless we identify additional labs to include                                                                                                                                                                    |
-| lab.date       | Date     | yyyy-mm-dd                                                 |                                                                                                                                                                                                                             |
-| lab.route      | Text     | V                                                          | *Always "V"  Veinous for Alliance sites                                                                                                                                                                                     |
-| lab.result     | Text     |                                                            | Several values are possible; integer, non-integer numeric, ranges indicated by alphanumeric text.  We'll use previous examples to essentially construct a dictionary.                                                       |
+| Field           | Format   | Constraints                                                | Notes/Questions                                                                                                   |
+|-----------------|----------|------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| timestamp       | TimeDate | yyyy-mm-dd hh:mm:ss.sss-hh:mm                              | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                                                |
+| visit_date      | Date     | yyyy-mm-dd                                                 | Effective date of score                                                                                           |
```

Comment:
I don't see `visit_date` in the example?

Reply:
adding...